### ISSUE TITLE
Official patches

### ISSUE THREAD

**Issue by jburrows82** [gamerforEA/Cauldron-Reloaded#6]
Did you build against the Cauldron #198 (Forge 1207) source code and include/apply the Cauldron base code #198 -> #212 patches? Just curious because there are performance improvements along with several bugfixes. I'm just curious.

Thank you for your time and contribution to the community!

**Comment by Dark-Arcana** [comment]
I would like to know this as well. I have the source from #198 and some of the patches up to 216. If this is part of the official branch with an update to internal Forge without any malicious modification to internal code, it would be worth using.

**Comment by spannerman79** [comment]
I am not a coder but during compile I notice that it uses "Special Souce" which is from Spigot.  That is what I assume is the cauldron base patches... I could be wrong though but that is what I surmize

**Comment by Dark-Arcana** [comment]
Cauldron = Forge + Spigot + patches and helper classes. Spigot may get pulled in, but that doesn't make it Cauldron. Cauldron is actually an assortment of patches that get applied at compile time, before a production ready installer or server jar is made available.

Basically what this means is that Special Source, or any other library or addon like Forge, is an outside element. It gets pulled in via the gradle build script and built/compiled against. Patching and updating Cauldron isn't the same as changing the versioning of an addon, like updating Forge. Though this is fairly necessary for an end user to continue to use Cauldron, as it has basically died while Forge mods require updated versions that old builds of Cauldron do not support.

What I wanted to know was which of the official patches were applied, which new patches were applied, and which base build was this fork based on. I used to be on the Cauldron team. I have some "unofficial" patches from the old maintainer that updates to build 216, which was never released. Would be good to know what base this is, because it looks like the most competent so far.

**Comment by spannerman79** [comment]
I agreee @Dark-Arcana re: most competent.

Well if you were indeed part of the dev team for Cauldron perhaps you can do a pull request and add in up to build 216?

**Comment by Dark-Arcana** [comment]
I was the community manager, not a developer. Blood did all of the development for Cauldron. I don't have the patches anymore, and that's why I was wondering what patches were applied here because I might be able to get some that are missing.

**Comment by Myrninvollo** [comment]
HI dark and spammer man and everyone.   well glad to see someone is trying to keep cualdron updated, now if I can just get it to build il be too see the code better but I keep getting fail when it tryst to build fml properties file,  will post a paste bin of log later when I'm at my comp.

**Comment by Dark-Arcana** [comment]
@Myrninvollo you should probably post a separate issue for that problem.

**Comment by Myrninvollo** [comment]
Very true I will :)

**Comment by TheBrain0110** [comment]
Yeah, things are getting confusing. There were the official patches, the semi-official patches by blood, now the unofficial patches by I-don't-know-who, and now this open repo, based on I-don't-know-what.

So as it stands there's this one supporting Forge 1388, and patch 221 on Yive's Mirror with Forge 1381. How to choose which one to go with...

**Comment by Cracks777** [comment]
Actually if you check the binpatches.pack.lzma in the jar and compare the differences from 221 there are over 300 changes, with several new patches,there are more patches in this one than any of the other versions out there, the file size is also even larger.

**Comment by TheBrain0110** [comment]
Check the binpatches in which jar? The Reloaded one here? Are you saying this one is built off of the 221 build on Yive?

Differences in what, from what?

**Comment by Cracks777** [comment]
the Reloaded version has more patches than the 212-221 version.

**Comment by Cracks777** [comment]
no i dont think that its build off the 221 version, but it does have more patches than 221

**Comment by Shybella** [comment]
The patches are we're not directly from Blood. Most of them came from MD_5's Stash Hub for Spigot. This is the same for Forge updates. Blood may have improved them better naively but that's a whole another topic.

**Comment by Shybella** [comment]
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/spigot/browse
